src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -252,3 +256,218 @@ def test_WINNF_FT_S_SCS_15(self,config_filename):
 
     # Check registration response
     self.assertEqual(response['response']['responseCode'],104)
+
+  def generate_SCS_17_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_17. """
+    # Create the configuration for short lived invalid client cert/key,wait timer
+
+    # Get current working directory
+    cwd = os.getcwd()
+
+    # testcases directory
+    testcases_dir = os.path.dirname(
+      os.path.abspath(inspect.getfile(inspect.currentframe())))
+
+    # Absolute path to the certs directory
+    path = os.path.join(testcases_dir, '../certs')
+
+    # Change directory to the certs directory
+    os.chdir(path)
+
+    # Build short lived certificate command
+    command = "./generate_short_lived_certs.sh  CBSD  short_lived_client " + " 2 "
```

Comment:
The cert should be generated as part of the test case, not as part of the default config. The function to generate the default config will only be run once, but you need to generate a short-lived cert every time.

Reply:
As per   suggestion moved the  creation of   certificate at beginning of the test case. A utility function to create the   certificates has been added to the SecurityTestCase class and is invoked   within the test cases.